packages/curve-ui-kit/src/features/user-profile/settings/theme-selector/ThemeSelector.tsx

```diff
@@ -0,0 +1,32 @@
+import { t } from '@lingui/macro'
+import ToggleButton from '@mui/material/ToggleButton'
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup'
+
+import { themes } from '@ui-kit/features/switch-theme/types'
+
+import { Label, Setting } from '../Setting'
+import useUserProfileStore from '../../store'
+
+export const ThemeSelector = () => {
+  const theme = useUserProfileStore((state) => state.theme)
+  const setTheme = useUserProfileStore((state) => state.setTheme)
+
+  return (
+    <Setting>
+      <Label>{t`Theme`}</Label>
+
+      <ToggleButtonGroup
```

Comment:
Duplicate of `ThemeSwitcherButtons`

Reply:
Same reasoning as for the 'duplicate' advanced theme component